src/js/views/admin/sidebar/clinician/clinician-sidebar_views.js

```diff
@@ -0,0 +1,226 @@
+import _ from 'underscore';
+import hbs from 'handlebars-inline-precompile';
+import Radio from 'backbone.radio';
+import { View } from 'marionette';
+
+import InputWatcherBehavior from 'js/behaviors/input-watcher';
+import Droplist from 'js/components/droplist';
+
+import intl from 'js/i18n';
+
+import 'sass/modules/buttons.scss';
+import 'sass/modules/forms.scss';
+import 'sass/modules/sidebar.scss';
+
+import ClinicianSidebarTemplate from './clinician-sidebar.hbs';
+import ClinicianNameTemplate from './clinician-name.hbs';
+import ClinicianEmailTemplate from './clinician-email.hbs';
+
+const RoleItemTemplate = hbs`<a{{#if isSelected}} class="is-selected"{{/if}}>{{matchText name query}} <span class="clinician__role">{{matchText short query}}</span></a>`;
+
+const RoleComponent = Droplist.extend({
```

Comment:
This is identical to the RoleComponent in clinicians-all_views.js. So we'll probably want to find a common place for this. Would something like `clinicians-actions_views.js` make sense?